api/app/signals/apps/email_integrations/models.py

```diff
@@ -0,0 +1,29 @@
+from django.contrib.gis.db import models
+
+
+class EmailTemplate(models.Model):
```

Comment:
I think the following fields are missing: `created_at`, `created_by`, `updated_at`, `updated_by`
Also how do we track changes to the templates in the database? The current implementation is using files stored in git so we have a commit history to track changes in the templates.

Reply:
Updated this, now we inherit from `CreatedUpdatedModel` and also I added an `created_by` field. I did not implement a `updated_by` field anywhere. I think in the future we should move to an application-wide audit log.

Reply:
Change tracking is not part of this implementation.

Reply:
For now tracking changes in the Django Admin is sufficient. However if we are going to make this editable through the API we need to make sure this is still done.

Also at this moment we have 2 types of "history" tracking solutions implemented. @tcoenen, @CBuiVNG and me already talked about making a more general solution for history tracking.